core/templates/pages/exploration-editor-page/services/change-list.service.spec.ts

```diff
@@ -333,18 +332,24 @@ describe('Change List Service when internet is available', () => {
         {
           provide: AutosaveInfoModalsService,
           useValue: mockAutosaveInfoModalsService
+        },
+        {
+          provide: InternetConnectivityService,
+          useValue: {
+            onInternetStateChange: onInternetStateChangeEventEmitter,
+            isOnline() {
+              return true;
+            }
+          }
         }
       ]
     });
   }));
 
   beforeEach(() => {
     changeListService = TestBed.inject(ChangeListService);
-    internetConnectivityService = TestBed.inject(InternetConnectivityService);
+    TestBed.inject(InternetConnectivityService);
```

Comment:
Do we still need to inject this? Since it's already being provided above.

Reply:
done